tests/ctests/test_netplan_keyfile.c

```diff
@@ -255,6 +255,71 @@ test_load_keyfile_route_options_without_route(__unused void** state)
     netplan_state_clear(&np_state);
 }
 
+void
+test_load_keyfile_utf8_password(__unused void** state)
+{
+    NetplanState *np_state = NULL;
+    int fd;
+    int size;
+    int res;
+    char* yaml;
+
+    const char* keyfile =
+        "[connection]\n"
+        "id=mywifi\n"
+        "uuid=03c8f2a7-268d-4765-b626-efcc02dd686c\n"
+        "type=wifi\n"
+        "interface-name=wlp2s0\n"
+        "[wifi]\n"
+        "mode=infrastructure\n"
+        "ssid=mywifi\n"
+        "[wifi-security]\n"
+        "key-mgmt=wpa-psk\n"
+        "password=áÁéÉíÍóÓúÚ\n"
+        "[ipv4]\n"
+        "method=auto\n";
+
+    const char* expected =
+        "network:\n"
+        "  version: 2\n"
+        "  wifis:\n"
+        "    NM-03c8f2a7-268d-4765-b626-efcc02dd686c:\n"
+        "      renderer: NetworkManager\n"
+        "      match:\n"
+        "        name: \"wlp2s0\"\n"
+        "      dhcp4: true\n"
+        "      access-points:\n"
+        "        \"mywifi\":\n"
+        "          auth:\n"
+        "            key-management: \"psk\"\n"
+        "          networkmanager:\n"
+        "            uuid: \"03c8f2a7-268d-4765-b626-efcc02dd686c\"\n"
+        "            name: \"mywifi\"\n"
+        "            passthrough:\n"
+        "              wifi-security.password: \"áÁéÉíÍóÓúÚ\"\n"
```

Comment:
todo: huh? This should be in "auth.password" instead... Not in passthrough, for a normal WPA-PSK

Reply:
oh god... I copied it from another test and reused the 802.1x section...